parsec/core/mountpoint/winfsp_runner.py

```diff
@@ -20,80 +22,56 @@
 logger = get_logger()
 
 
-async def cleanup_broken_links(path: trio.Path) -> None:
-    def target():
-        for child in sync_path.iterdir():
-            try:
-                if not child.exists():
-                    child.unlink()
-            except OSError:
-                pass
-
-    # This should be migrated to trio.Path API once issue #1308 is fixed.
-    sync_path = Path(str(path))
-    await trio.to_thread.run_sync(target)
-
-
-async def is_path_available(path: trio.Path) -> bool:
-    # The path already exists
-    if await path.exists():
-        return False
-
-    # The path is a broken link
-    try:
-        await path.lstat()
-        return True
-    except OSError:
-        pass
-
-    # The path is available
-    return True
-
-
-async def _bootstrap_mountpoint(base_mountpoint_path: Path, workspace_name) -> Path:
-    # Mountpoint can be a drive letter, in such case nothing to do
-    if str(base_mountpoint_path) == base_mountpoint_path.drive:
-        return
-
-    # On Windows, only mountpoint's parent must exists
-    trio_base_mountpoint_path = trio.Path(base_mountpoint_path)
-    await trio_base_mountpoint_path.mkdir(exist_ok=True, parents=True)
+BASE = "PQRSTUVWXYZHIJKLMNO"
+assert len(BASE) == 19
```

Comment:
Why 19 is so important ? (I guess it's for the algo in `candidates`, but would be good to document this)

Reply:
Cause it's prime, but a comment is definitely need :+1: